gruntfile.js

```diff
@@ -388,7 +280,7 @@ module.exports = function (grunt) {
       },
       build: {
         files: ['<%= src.js %>', '<%= src.css %>', '<%= src.tpl %>', '<%= src.html %>'],
-        tasks: ['build', 'shell:buildImage', 'shell:run', 'shell:cleanImages']
+        tasks: ['watchBuild:']
```

Comment:
IMO we should only be able to trigger a build with watch for the local endpoint. No need for the SSL or Swarm setups at the moment since we can still add them as endpoints in the UI.

You can remove anything related to runSSL, runSwarm, buildSSL, etc...